### DISCUSSION TITLE
Which and What do I have to Edit to Share with Gradio?

### DISCUSSION THREAD

**Discussion by junior-2050** [lllyasviel/Fooocus#3055] [Q&A] [answered]
In the description is says this:

In addition to running on localhost, Fooocus can also expose its UI in two ways:
    Local UI listener: use --listen (specify port e.g. with --port 8888).
    API access: use --share (registers an endpoint at .gradio.live).
    
However in the Python Script when it is Loaded it says this:

    Running on local URL:  http://127.0.0.1:7865

    To create a public link, set `share=True` in `launch()`.

I have Added so many lines in various places, searched for *share=* in the webui.py, shared.py, and launch.py. However NOTHING is working I just get script errors. I just want to have Localized Network access from my other computer. All of the Google searched references the older Stable Diffusion which is an older version of this and Does nothing to enable the Share Mode using Gradio. 

Does anyone know WHICH file, which Line, and what I have to add to enable this option?

**Comment by Rogala** [comment, marked as answer]
To get the link from the gradio, all you have to do is type in run.bat `--share`

`.\python_embeded\python.exe -s Fooocus\entry_with_update.py --share`
